### PR TITLE
`treehouses message` formatting (fixes #2050)

### LINKED ISSUE #2050: `treehouses message gitter` inside of `README.md` should be formatted correctly
The `message gitter` inside of `README.md` should be separated (there is no stand-alone command for `treehouses message gitter`)  to its own line.

https://github.com/treehouses/cli/blob/92210649dbb3473185d691006793f8cdb0bbd30d/README.md#L168

---

`message` should be in its own stand-alone line.

---

The subcommands for `message`:
* gitter
* slack
* discord

should all be wrapped by `<>` brackets.

https://github.com/treehouses/cli/blob/92210649dbb3473185d691006793f8cdb0bbd30d/README.md#L168-L173

The `magazine` subcommand from Lines 159 to 165 should be the ideal format for the `message` subcommand.

https://github.com/treehouses/cli/blob/92210649dbb3473185d691006793f8cdb0bbd30d/README.md#L159-L165

### PR DESCRIPTION
**Pull request by rjpadilla**
Fixed the `message` command to its correct format.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,12 +165,13 @@ magazines                                 downloads specific magazine issue as a
...
 resolution <cea|dmt [modes]>              sets the screen resolution
 system [cpu|ram|disk|volt|temperature]    display real system informations
-message gitter <apitoken|authorize>       sets api/channel info in config file and sends/recieves messages in gitter
-               <send|show|read|mark>
-               <channels>
-        slack <apitoken|channels>         sets api/channel info in config file and sends/recieves messages in slack
-              <send|show|read|mark>
-        discord <apitoken|authorize>      sets api/channel info in config file
+message                                   sends message to chat service
+   <gitter>  <apitoken|authorize>         sets api/channel info in config file and sends/recieves messages in gitter
```
Comment (dogi) [inline README.md:169]: split comment line

File: README.md [hunk 2]
```diff
@@ -165,12 +165,13 @@ magazines                                 downloads specific magazine issue as a
...
+             <send|show|read|mark>
+             <channels>
+   <slack>   <apitoken|channels>          sets api/channel info in config file and sends/recieves messages in slack
```
Comment (dogi) [inline README.md:172]: split comment line

File: README.md [hunk 3]
```diff
@@ -165,12 +165,13 @@ magazines                                 downloads specific magazine issue as a
...
+             <send|show|read|mark>
+   <discord> <apitoken|authorize>         sets api/channel info in config file
```
Comment (dogi) [inline README.md:174]: every change in README.md should also be in help.sh

**Review by dogi — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (JLKwong) [reply README.md:174]: The new format for the subcommands should also be included in the message.sh file at the end (message_help function).

**Review by JLKwong — CHANGES_REQUESTED** [review]
The new format for the subcommands should also be included in the message.sh file at the end (message_help function).

File: README.md (hunk 3 above)
Comment (rjpadilla) [reply README.md]: > 
> 
> The new format for the subcommands should also be included in the message.sh file at the end (message_help function).

Yah....
I should have just made a single issue to fix both (see #2049)

**Comment by rjpadilla** [comment]
> 
> 
> The new format for the subcommands should also be included in the message.sh file at the end (message_help function).

**Review by dogi — APPROVED** [review]

**Pull request merged**